evidence/reactor.go

```diff
@@ -252,3 +254,7 @@ func evidenceListFromProto(m proto.Message) ([]types.Evidence, error) {
 
 	return evis, nil
 }
+
+func (evR *Reactor) OnStop() {
+	evR.BaseReactor.OnStop()
+}
```

Comment:
this feels redundant since OnStop() is already called automatically when stopping a reactor